src/css/volunteers-list.css

```
@@ -0,0 +1,735 @@
+@reference "../app/globals.css";
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@reference" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)